eip-1271/nodejs/README.md

```diff
@@ -0,0 +1,200 @@
+   - Because the `WhitelistEIP1271` contract was deployed with a signature threshold of `2`, after two signatures from whitelisted addresses are provided, the EIP-1271 function `isValidSignature` will return the magic value (`0x1626ba7e`) signalling the signature as valid.
+9. Calls `litNodeClient.createCapacityDelegationAuthSig` to generate a `capacityDelegationAuthSig` to pay for the execution of `litNodeClient.getPkpSessionSigs`
+10. Calls `litNodeClient.getPkpSessionSigs` to generate Session Sigs using the previously minted PKP
+    - This steps provides the EIP-1271 Auth Lit Action code ([litAction.ts](./src/litAction.ts)) as a [code string](https://developer.litprotocol.com/sdk/serverless-signing/deploying#deploying-using-a-code-string), and when the Lit nodes receive the request to generate their Session Sig shares, they will:
+      1. Compute the IPFS CID for the given Lit Action code string
+      2. Request the permitted Auth Methods for the PKP from the Chronicle Yellowstone Blockchain
+      3. Verify the IPFS CID is one of the permitted Auth Methods for the PKP
+      4. Execute the given Lit Action code
+         - The Lit Action will:
+           - Call the `isValidSignature` method on the `WhitelistEIP1271` contract to check if the provided EIP-1271 signature is valid
+           - If valid, returns true signalling the Lit node to generate it's Session Signature share
+
+## How to Run the Example
+
+### Step 1: Clone the Code Example
+
+Clone the Developer Guides Code repo using:
+
+```
+git clone https://github.com/LIT-Protocol/developer-guides-code.git
+```
+
+You'll want to `cd` into the code example directory:
+
+```
+cd eip-1271
+```
+
+Here you will see two directories:
+
+1. `contracts` This is a [Foundry](https://github.com/foundry-rs/foundry) project for the `WhitelistEIP1271` contract
+2. `nodejs` This is the directory for the code example
+
+### Step 2: Deploy the `WhitelistEIP1271` Contract
+
+If you don't already have Foundry installed, you can do so by following [this](https://book.getfoundry.sh/getting-started/installation) guide.
+
+Now `cd` into the `contracts` directory:
+
+```
+cd contracts
+```
+
+And install the project using:
+
+```
+forge install
+```
+
+Because the Lit nodes will make an RPC call to a blockchain to call the `isValidSignature` on the `WhitelistEIP1271` contract when running the Lit Action, and they don't have access to your `localhost`, we must deploy the `WhitelistEIP1271` to a public blockchain like the Lit Chronicle Yellowstone blockchain.
+
+To deploy the contract to Chronicle Yellowstone, you'll need Lit test tokens. If you don't have any, you can request some using a [faucet](https://chronicle-yellowstone-faucet.getlit.dev/).
+
+Deploy the contract to Chronicle Yellowstone using:
+
+```
+PRIVATE_KEY=YOUR_PRIVATE_KEY forge script script/WhitelistEIP1271.s.sol:DeployWhitelistEIP1271 --rpc-url https://yellowstone-rpc.litprotocol.com/ --broadcast
+```
+
```

Comment:
```suggestion
Replace `YOUR_PRIVATE_KEY` here with your Ethereum private key.
```

This is pretty obvious to do, but making sure the user knows might be good. Either way they can probably figure this out